catalyst-gateway/bin/src/service/api/cardano/rbac/registrations_get/invalid_registration.rs

```diff
@@ -60,7 +66,7 @@ impl From<Query> for InvalidRegistration {
             txn_index: q.txn_index.into(),
             previous_txn: q.prv_txn_id.map(|t| TransactionId::from(t).into()),
             purpose: q.purpose.map(|p| UuidV4::from(p).into()),
-            report: q.problem_report.into(),
+            report: ParseFromJSON::parse_from_json_string(&q.problem_report).unwrap_or_default(),
```

Comment:
Also change it to `TryFrom` trait implementation 
```suggestion
            report: ParseFromJSON::parse_from_json_string(&q.problem_report)?,
```